mail-filter/sqlgrey/sqlgrey-1.8.0.ebuild

```diff
@@ -0,0 +1,246 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd
+
+DESCRIPTION="SQLgrey is a postfix policy service implementing a grey-listing policy"
```

Comment:
I've committed an initial EAPI 7 bump as de2c9c7cc5c8c1e28fe64de78283650e16e3b7af. Please make matching changes in your ebuild.